generators/graph/voxel_generator_graph.cpp

```diff
@@ -411,7 +411,7 @@ void VoxelGeneratorGraph::load_graph_from_variant_data(Dictionary data) {
 	const VoxelGraphNodeDB &type_db = *VoxelGraphNodeDB::get_singleton();
 
 	const Variant *id_key = nullptr;
-	while (id_key = nodes_data.next(id_key)) {
+	while ((id_key = nodes_data.next(id_key))) {
```

Comment:
@Zylann Thats the Wparentheses warning. I saw something similar is previous warning fix PR, but can't see it right now. My understanding is that because we are assigning  `id_key`, not directly equating, gcc wants the parentheses to show that it is in fact a conditional.

Reply:
I don't relate to this tbh. It's just completely redundant. I've seen this warning being of importance when operator precedence can be misunderstood but that doesnt seem to be the case here...